src/components/soknadsStatus/soknadsStatusUtils.ts

```diff
@@ -18,3 +18,5 @@ const soknadsStatusTittel = (status: SoknadsStatusResponseStatus | undefined, t:
 };
 
 export {soknadsStatusTittel};
+
+export const DEFAULT_SAK_TITTEL = "default_sak_tittel";
```

Comment:
trenger man egentlig en egen konstant for denne?

Reply:
Neei... Endrer.